vita3k/ctrl/src/ctrl.cpp

```diff
@@ -258,6 +258,11 @@ static void apply_controller(EmuEnvState &emuenv, uint32_t *buttons, float axes[
 static void retrieve_ctrl_data(EmuEnvState &emuenv, int port, bool is_v2, bool negative, bool from_ext_function, SceUInt32 &buttons, SceUInt8 &lx, SceUInt8 &ly, SceUInt8 &rx, SceUInt8 &ry) {
     std::lock_guard<std::mutex> guard(emuenv.ctrl.mutex);
 
+    if(emuenv.ctrl.drop_inputs){
```

Comment:
Just moving it with other if in 285 line, no have sense here

Reply:
Could you tell me what `SCE_COMMON_DIALOG_RUNNING` does so that i have more information as to if it is an appropriate checking point or not?

Reply:
? Just read code

Reply:
Zang, I am asking you because I do not understand what the code is telling me. There is no documentation, no comments, no nothing for that enum and it's constants. I am grasping at straws. Best I can tell is that it's supposed to be for if an in-game dialogue is pulled up, which WOULD make sense to drop all inputs for, so I could `OR` it with my check, but I have asked for information from you because I am not sure, and would like to be certain. It won't kill you to help someone out.

Reply:
I said that because there was nothing to say, just read the code, it is clear and clear, no need for comments, and very simple to understand.  if the commong dialog is in progress it deactivates the ingame input, easy to understand